### ISSUE TITLE
Committer application

### ISSUE THREAD

**Issue by LouisAUTHIE** [PrestaShop/open-source#217]
Hi everyone,

Regarding the slow delays of many pull request approvals, I'd like to help to improve the core of PS.

Thanks

**Comment by matks** [comment]
Hey I'm really happy to see that GitHub issue 😉 , however I think this application is a little bit short 😅 . I can suggest reading other examples of applications (see below) like the idea is to demonstrate to team members that you can be trusted to review PRs.

I know you so I have a good idea of your skills 😊  but I'm not the only one to decide

Examples
https://github.com/PrestaShop/open-source/issues/106
https://github.com/PrestaShop/open-source/issues/105
https://github.com/PrestaShop/open-source/issues/103

Very simple example: add this to your message 😋 https://github.com/pulls?q=is%3Apr+author%3Aokom3pom+archived%3Afalse+org%3Aprestashop+author%3ALouisAUTHIE

Basically when you say "I would like to become a committer" it's a little bit "Can you please give me a copy of your keys" so people will want to be reassured 😄

**Comment by LouisAUTHIE** [comment]
Speed wouldn't be the only drive, just to add some explanation to my previous message. But I really feed that we need to find a good ratio speed/quality if we want to finish the migration quickly.

**Comment by matks** [comment]
Fun fact 😄 this is the 2nd application to this project motivated by "it's too long to review"

First one was https://github.com/PrestaShop/open-source/issues/48

**Comment by boherm** [comment]
Be sure that's we want on our side too! :)

**Comment by LouisAUTHIE** [comment]
My resume : 
I'm a freelance developer working with php, swift, java, etc...
I work with PS since 2012 and I have some PRs even if I'm a little contributor : 
https://github.com/PrestaShop/PrestaShop/commits?author=LouisAUTHIE
and https://github.com/PrestaShop/PrestaShop/commits?author=WebHelpersPau

I have written the "Practical modules development for PrestaShop 8" book to make people follow the Symfony changes.
Hope to help you make things improve

**Comment by Hlavtox** [comment]
Not many PRs from Louis, but those about store commands migration show that he definitely got the skills. :-)

**Comment by LouisAUTHIE** [comment]
Don't take my "it's too long" application as a rude thing, I just put myself as the normal contributor with a busy like as everyone and I try to help with my time and quality standards. I am not looking for the perfect software, I am looking for the appropriate level of state of the art features together with the security/maintainability level.

**Comment by LouisAUTHIE** [comment]
> Be sure that's we want on our side too! :)

I am convinced about it, and thanks again for your work. Anyway we need to make things improve to encourage the little contributors to feel me efficient.

**Comment by okom3pom** [comment]
Suspect url :) @matks https://github.com/pulls?q=is%3Apr+author%3Aokom3pom+archived%3Afalse+org%3Aprestashop+author%3ALouisAUTHIE i see my name 🤗

**Comment by jf-viguier** [comment]
Louis will be a very good committer, I hope he'll be acceted at the PrestaShop'CIA

**Comment by PrestaEdit** [comment]
Yeah, maybe you can follow this one : https://github.com/PrestaShop/open-source/issues/198

You know, having somes datas inside your initial post.
Or maybe be part of PrestaShop SA :trollface:

**Comment by matks** [comment]
> Suspect url :) @matks https://github.com/pulls?q=is%3Apr+author%3Aokom3pom+archived%3Afalse+org%3Aprestashop+author%3ALouisAUTHIE i see my name 🤗

Because I copy-pasted the link from your past application 😄 yeah I was a bit lazy to redo it from scratch

> Yeah, maybe you can follow this one : #198
> 
> You know, having somes datas inside your initial post. Or maybe be part of PrestaShop SA :trollface:

Hey @PrestaEdit I'm sad to see you trolling about this topic. And on top of this you're ruining this application (you could have chosen another place than @LouisAUTHIE application).

I really hope one day you'll be able to get over this stuff ❤️ . It seems half of the things on this GitHub org you say is just hate speech, troll, nasty messages against PrestaShop SA. Always the same sad song. It obviously brings you no benefit, but it helps keeping alive a somehow sordid atmosphere (and give you a bad image). Have you ever considered that by doing this, not only are you hurting people's feeling, you're hurting yours as well? Having negative thoughts is bad for both body and mental health.

I hope this sorrow and hate in your heart could go away and you could find peace 😊 free of negative feelings.

**Comment by PrestaEdit** [comment]
Oh, you know... my feelings are not about PrestaShop SA, but more about PrestaShop "open-source", but as ever and ever, you don't read anything else that exists, only what could usefull to yours feeling ;-)

Anyway, my bad to @LouisAUTHIE - the only one needed my excuses on this side.

And, yeah, 😽 to everyone reading this ! :)

**Comment by LouisAUTHIE** [comment]
About my free time, I won’t have as much time as a fully dedicated reviewer but my aim is to give some hours a week to do it. We’ll see how to works.

**Comment by LouisAUTHIE** [comment]
> @PrestaEdit with a technical interview behind you, a contract requiring you to work on the software for the next X months full-time, and a number of approved, merged PRs - but hey, no, it's just about being in PrestaShop SA! 🤡
> 
> @LouisAUTHIE I would love to vote yes, so we can see if you can help us in the next 90 days during the test period 👍🏻 you have the skills, not sure if you are willing to invest time to do reviews, but if you are motivated to do so... :)

Let's give it a try, we will see then if it helps. The first point there is to see it the technical level is ok with the project standards

**Comment by LouisAUTHIE** [comment]
> Oh, you know... my feelings are not about PrestaShop SA, but more about PrestaShop "open-source", but as ever and ever, you don't read anything else that exists, only what could usefull to yours feeling ;-)
> 
> Anyway, my bad to @LouisAUTHIE - the only one needed my excuses on this side.
> 
> And, yeah, 😽 to everyone reading this ! :)

No worries, I don't really care about these "political" jokes !!!
Enjoy

**Comment by matks** [comment]
Going back on the initial topic. The discussion diverted into other matters but we owe a solid discussion based on his application.

First of all I'm going to quote our rules: https://www.prestashop-project.org/maintainers-guide/how-to-become-a-committer/
> If three maintainers approve the application, the new committer is approved.

These rules need to be updated 🤔 as the concept of "maintainers" was removed in favor of "technical council" https://www.prestashop-project.org/project-organization/people-and-roles/ but the idea is that 3 people from the following group must approve your application to validate it

Here are the people:
-  Jonathan Lelièvre ([@jolelievre](https://github.com/jolelievre)) - Council Lead
- Daniel Hlavacek ([@Hlavtox](https://github.com/Hlavtox))
- Ibrahima Sow ([@sowbiba](https://github.com/sowbiba))
- Krystian Podemski ([@kpodemski](https://github.com/kpodemski))
- myself ([@matks](https://github.com/matks))
- Matthieu Rolland ([@matthieu-rolland](https://github.com/matthieu-rolland))
- Pablo Borowicz ([@eternoendless](https://github.com/eternoendless))
- Pululu Kinanga André ([@PululuK](https://github.com/PululuK))
- Thomas Baccelli ([@atomiix](https://github.com/atomiix))
- Valentin Szczupak ([@NeOMakinG](https://github.com/NeOMakinG))

@Hlavtox I think your message is an approval.

After reading again https://build.prestashop-project.org/news/2023/contributor-interview-louis-authie/ (yes I remember 😉 ) I add my Approval: so that makes two! One more and you're accepted!

While waiting for another technical council member to review your application (this can take days of weeks, especially in summer so please be patient), do you have any questions?

For the record here's a very interesting take about becoming a committer https://us11.campaign-archive.com/?e=2323a8e4c7&u=9d7ced8c4bbd6c2f238673f0f&id=d6f2c9f242

**Comment by matthieu-rolland** [comment]
> @LouisAUTHIE I would love to vote yes, so we can see if you can help us in the next 90 days during the test period 👍🏻 you have the skills, not sure if you are willing to invest time to do reviews, but if you are motivated to do so... :)

Same here 

it's a yes for me :+1:

**Comment by jf-viguier** [comment]
Bravo Louis
